pages/community/ambassador-program/index.js

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+import Button from '../../../components/buttons/Button';
+import GenericLayout from '../../../components/layout/GenericLayout';
+import Heading from '../../../components/typography/Heading';
+import IconRocket from '../../../components/icons/Rocket';
+import NewsletterSubscribe from '../../../components/NewsletterSubscribe';
+import ambassadors from '../../../config/AMBASSADORS_MEMBERS.json';
+
+const data = [
+  {
+    title: 'Written content',
+    details:
+      'Write guides, step-by-step tutorials, or best practice cheat sheets for the AsyncAPI blog or your own.',
+    icon: '/img/illustrations/blog.svg',
+  },
+  {
+    title: 'Video content',
+    details:
+      'Produce educational videos or AsyncAPI developer training for AsyncAPI’s YouTube channel or your own.',
+    icon: '/img/illustrations/video-creation.svg',
+  },
+  {
+    title: 'Live streams',
+    details:
+      'Moderate or host videos and live streams that demo AsyncAPI and promote the ecosystem.',
+    icon: '/img/illustrations/live.svg',
+  },
+  {
+    title: 'Give talks',
+    details:
+      'Speak at meetups and conferences, and we’ll help with slides, abstract submissions, and travel budget.',
+    icon: '/img/illustrations/speaking.svg',
+  },
+  {
+    title: 'Learning apps',
+    details:
+      'Build educational applications and games to teach developers about AsycnAPI and event driven architectures.',
+    icon: '/img/illustrations/learning-app.svg',
+  },
+  {
+    title: 'Build educational apps',
+    details: 'Develop applications for hands-on learning in any language.',
+    icon: '/img/illustrations/codes.svg',
+  },
+  {
+    title: 'AsyncAPI Advisor',
+    details:
+      'Collaborate with the AsyncAPI team to boost awareness of open source.',
+    icon: '/img/illustrations/advisor.svg',
+  },
+  {
+    title: 'Gather Usecases',
+    details: 'Collect data from existing AsyncAPI users ',
+    icon: '/img/illustrations/meeting.jpg',
+  },
+];
+
+
+const tokens = [
+  {
+    emoji: '📚',
+    title: 'Education',
+    details:
+      'Ambassadors are provided with a budget plan that they use for personal development tools, like books, courses, and other learning methods.',
```

Comment:
not true 😄

Reply:
I thought you said privately that there would be a budget for ambassadors' education 😆. Guess i heard you wrong

Reply:
well maybe in future, once we have some payed education 😄 but for now it is not mentioned in official documentation